src/components/DownloadButton/index.tsx

```diff
@@ -9,7 +9,7 @@ import { logEvent } from '@dvcorg/gatsby-theme-iterative/src/utils/front/plausib
 import * as styles from './styles.module.css'
 import { OS, useUserOS } from '../../utils/front/useUserOS'
 
-const VERSION = `2.30.0`
+const VERSION = `2.29.0`
```

Comment:
Not sure how this happened during some merge from `main`. Reverting...
```suggestion
const VERSION = `2.30.0`
```